src/store/getters.js

```diff
@@ -132,6 +132,12 @@ export default {
    * Get map of active filters
    */
   activeFilters: state => state.filters.selections,
+  filterLoadingDict: state => state.filterLoadingDict,
```

Comment:
please do not make these passthrough getters, just use the mapState[''] function in the computed section of the component

This just adds more code to test, which has no logic.

Reply:
removed the getters but kept "activeFilters" as it is (i i dont think that i wrote this getter...)